js/index.js

```diff
@@ -38,5 +40,94 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
+
+//nav selector
+const [ nav ] = document.getElementsByTagName('nav');
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// CTA Selector
+const [ cta ] = document.getElementsByClassName('cta');
+const [ ctaH1 ] = document.getElementsByTagName('h1');
+const [ ctaBtn ] = document.getElementsByTagName('button');
+const ctaImg = document.getElementById('cta-img');
+
+//Main Content Selector
+const [ mc ] = document.getElementsByClassName('main-content');
+
+//Middle Image Selector
+const middleImg = document.getElementById('middle-img');
+
+//Bottom Content Selector
+const [ bc ] = document.getElementsByClassName("bottom-content");
+
+// Text Content H4 and P Content Selectors
+const textContentH4 = document.querySelectorAll(".text-content h4");
+const textContentP = document.querySelectorAll(".text-content p");
+
+// Contact H4 and P Content Selectors
+const contactH4 = document.querySelectorAll(".contact h4");
+const contactP = document.querySelectorAll(".contact p");
+
+// Footer
+const footerP = document.querySelectorAll("footer p");
```

Comment:
I like how you're keeping your selectors at the top of the file here - nice and organised. 👌